FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 2
+title: MEV and Liquidations
+team: Kaihua Qin, Liyi Zhou, Arthur Gervais
+created: 2020-12-10
+paper: 1
+---
+
+# Summary
+With over 6B USD total value locked for the top 3 lending protocols alone, this use case has grown into a substantial DeFi market. Lending entails the risks of a diminishing collateral value, which may render debt unhealthy. Unhealthy debt is typically resolved through liquidations, a process of selling collateral at a discount to liquidators.
+
+One dominant liquidation design is the fixed spread liquidation mechanism, used by Aave, Compound and dYdX. Interestingly, the fraction of the collateral that a liquidator can purchase at a discount amounts to 50% for Aave and Compound, and even 100% for dYdX (commonly referred to as close factor). It therefore appears that the fixed spread liquidation mechanisms are benefitting the liquidators more than the borrowers.
```

Comment:
What other mechanisms are there?  Pointer to those?

Reply:
In the wild, I'm aware of fixed spread liquidations (Aave, dYdX, Compound) and auctions (MakerDAO). Research wise, especially in the auction systems, there could be a much wider variety given the years of related work in TradFi.